arex-storage-web-api/src/main/java/com/arextest/storage/utils/DatabaseUtils.java

```diff
@@ -30,9 +36,9 @@ public class DatabaseUtils {
     private DatabaseUtils() {
```

Comment:
what is the purpose of it

Reply:
sonarqube rule: Hide Utility Class Constructor